src/widget/wstarrating.cpp

```diff
@@ -6,7 +6,8 @@
 #include "widget/wstarrating.h"
 
 WStarRating::WStarRating(QString group, QWidget* pParent)
-        : WBaseWidget(pParent),
+        : QWidget(pParent),
+          WBaseWidget(this),
```

Comment:
This is what I did in the old PR https://github.com/mixxxdj/mixxx/pull/613 but @rryan noticed it and I thought I was wrong. Let's do it again! \o/